### ISSUE TITLE
[YouTube] Very slow downloading speed

### ISSUE THREAD

**Issue by BlackMightyRavenDark** [ytdl-org/youtube-dl#32863]
Hi! Sorry for incomlpete template. But this issue is very simple to be described.
Some days ago i've expected a very slow and unstable downloading speed.
My log:
```
[youtube] 7szcXCT-Oqw: Downloading webpage
[dashsegments] Total fragments: 697
[download] Destination: Magical Underwater World 4K-7szcXCT-Oqw.f313.webm
[download]   0.6% of ~6.81GiB at 436.76KiB/s ETA 04:30:43
ERROR: Interrupted by user
```
Actually, the real speed is less than 100 KB/sec. All other downloading software has the same issue.
However, youtube is working fine in my browser.

**Comment by dirkf** [comment]
It's always better to provide the full verbose log as the template requires. What may look like dross at the start is necessary to diagnose problems.

In this case you should also tell us about your network connection, especially anything unusual, eg, VPN, state censorship firewall (oh, not so unusual).

>... All other downloading software has the same issue. ...

With YT or other sites too?

>.... youtube is working fine in my browser.

YT only has to send the video somewhat faster than real-time and not even that (or perhaps not at all) if your client is fingered as not being the YT player JS.

This is my initial result, with a domestic ISP ADSL2+ connection (not the limit here). Maybe the later GB would go slower, but I only checked the initial speed:
```console
$ youtube-dl-20240711 -v -f 313 7szcXCT-Oqw
[debug] System config: [u'--prefer-ffmpeg']
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: [u'-v', u'-f', u'313', u'7szcXCT-Oqw']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2024.07.11 [16f5bbc46] (single file build)
[debug] ** This version was built from the latest master code at https://github.com/ytdl-org/youtube-dl.
[debug] ** For support, visit the main site.
[debug] Python 2.7.18 (CPython i686 32bit) - Linux-4.4.0-210-generic-i686-with-Ubuntu-16.04-xenial - OpenSSL 1.1.1w  11 Sep 2023 - glibc 2.15
[debug] exe versions: avconv 4.3, avprobe 4.3, ffmpeg 4.3, ffprobe 4.3
[debug] Proxy map: {}
[youtube] 7szcXCT-Oqw: Downloading webpage
[debug] [youtube] Decrypted nsig hxup9EM64z724nVX => AlNJQM3ptGoezQ
[debug] [youtube] Decrypted nsig hrrC5Yv65DjNozWM => 8mD4kp_Ll7cI9Q
[debug] Invoking downloader on u'https://rr3---sn-cu-aigss.googlevideo.com/videoplayback?sparams=expire%2Cei%2Cip%2Cid%2Caitags%2Csource%2Crequiressl%2Cxpc%2Cbui%2Cspc%2Cvprv%2Csvpuc%2Cmime%2Cns%2Crqh%2Cgir%2Cclen%2Cdur%2Clmt&ei=h5ibZrGkMLXBmLAPtdOO6Ak&ip=46.208.6.101&clen=7304448660&spc=NO7bAaK4mk6CzdthMLrdB2h2JQCCHd9k11KGcxcecVBYBVpZvIHDIXOuovpc&id=o-ABe3W3XpZ6zBZMz1JI75DQmFExI8NkgOVm7rQdcvuwtZ&txp=5532432&svpuc=1&aitags=133%2C134%2C135%2C136%2C137%2C160%2C242%2C243%2C244%2C247%2C248%2C271%2C278%2C313%2C394%2C395%2C396%2C397%2C398%2C399%2C400%2C401&gir=yes&xpc=EgVo2aDSNQ%3D%3D&requiressl=yes&keepalive=yes&source=youtube&mv=m&sig=AJfQdSswRQIhAKWmODAdZ_riLzgkdfB_9OK1j0yAfyYsq38YzKuvZTBxAiBVNSHsjq0zJFLHsFMfOUCTZ0Dq6zB4IxNgktJT_DAECw%3D%3D&dur=3641.083&ns=aDg1_mXZ48bDuaWTZNaGOpEQ&initcwndbps=1762500&vprv=1&lsig=AHlkHjAwRAIgSHcPr4xbOcSWqiv8lI6uCyBQl--DYzVEn1UEqh5Vw1MCIE20Rh8tjvbTgIFsKq2_SF7MByX1JEA25gUh_C8Tr5gB&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lmt=1620072065017140&c=WEB&sefc=1&bui=AXc671J8ZhXIn8FlvjoC1aqlL1z0RjXE30dj3H-Wh542JCN_fWbp9XZLUGkXCFQgaQUKfJEMwXDnNPJs&mime=video%2Fwebm&fvip=5&rqh=1&itag=313&mm=31%2C29&mn=sn-cu-aigss%2Csn-cu-c9i6&mh=0h&n=8mD4kp_Ll7cI9Q&mt=1721472770&expire=1721494759&pl=25&ms=au%2Crdu&mvi=3'
[dashsegments] Total fragments: 697
[download] Destination: Magical Underwater World 4K-7szcXCT-Oqw.webm
[download]   0.4% of ~6.81GiB at  1.02MiB/s ETA 01:53:39
^C
ERROR: Interrupted by user
$
```

**Comment by BlackMightyRavenDark** [comment]
> In this case you should also tell us about your network connection, especially anything unusual, eg, VPN, state censorship firewall (oh, not so unusual).

I'm using 100 MBit/s direct connection with disabled firewall while tests. There is no YT blocking or censorship in my country. Do you need some more info?

> With YT or other sites too?

Sorry. I didn't understand your question. Can you rephrase it?
If you talking about Twitch etc. - yes, it's working fine.

**Comment by dirkf** [comment]
Yes, I was asking if other sites are downloading slowly (a) with _yt-dl_ (b) with any other tool. Anyhow, the [recently reported Russian throttling](https://github.com/yt-dlp/yt-dlp/issues/10443) isn't the issue.

Maybe your 100Mbit/s connection was too greedy and your IP address is being throttled by YT? You might, if possible, try some other connection as a test, via a mobile hotspot or your neighbour's wifi, say. Or switch between IPv4 and IPv6. Or ignore YT for a few days.

For any further help a verbose log will be required.

**Comment by BlackMightyRavenDark** [comment]
> [recently reported Russian throttling](https://github.com/yt-dlp/yt-dlp/issues/10443)

Oh, thanx so much! `goodbyedpi.exe -6 --frag-by-sni` is solved my problem.

**Comment by dirkf** [comment]
So something is deep interfering with your packets on the way between your system  and the YT servers. It could help other users if you were able to say what interference that is. Eg, is your ISP throttling YT traffic? Otherwise, the issue can be closed, I think.

**Comment by rocs1** [comment]
я в шоке

**Comment by dirkf** [comment]
Yes, shocked, I say, shocked.
